README.md

```diff
@@ -44,21 +44,23 @@ Orta Therox ([@orta](https://github.com/orta)), Sean Poulter ([@seanpoulter](htt
 * Highlights the errors next to the `expect` functions.
 * Adds syntax highlighting to snapshot files.
 * A one button update for failed snapshots.
-* Show coverage information in files being tested. (_requires coverage to be collected by your jest config_)
+* Show coverage information in files being tested. 
+* Help debug jest tests in vscode.
 
 ## How to get it?
 
 Open up VS Code, go search for the extension "Jest".  
 As an alternative, install with 1-click through [Visual Studio Marketplace - Orta.vscode-jest](https://marketplace.visualstudio.com/items?itemName=Orta.vscode-jest).
+ 
+The detailed releases and migration help: see [releases](https://github.com/jest-community/vscode-jest/releases).
 
 ## How to get it set up?
 
-This project has the expectation that you would run something like `npm run test` which _just_ looks like `jest` in the `package.json`. So, please keep your configuration inside the `package.json` as opposed to using command line arguments.
```

Comment:
What about "This extension runs on top of your Jest installation. When being started, it has the expectation that the Jest environment is properly set up, i.e. that the tests can be executed in VS Code's shell.